app/models/resource.rb

```diff
@@ -56,6 +56,7 @@ def latest(is_published = [true, false])
                        on translations.version = max_table.max_version
                        and translations.language_id = max_table.language_id
                        and translations.resource_id = max_table.resource_id")
+               .includes(:language).order('languages.name ASC')
```

Comment:
'ASC' is the default, you don't need to specify it.

Reply:
Yea thanks, I just like to be specific 😄